### DISCUSSION TITLE
WEC-Sim with MoorDyn

### DISCUSSION THREAD

**Discussion by eduguzman1996** [WEC-Sim/WEC-Sim#1258] [Q&A] [answered]
Is it possible to analyze an imbulical cable, using a pre-established floater motion.?
I would like to input the motion of the floater from an OpenFAST simulation. then i want to analyze the tension of the cable. 

Thank you !!

**Comment by jtgrasb** [comment, marked as answer]
Hi @eduguzman1996, thank you for your question. If you are using OpenFAST to solve the dynamics and determine the floater motion, you do not need WEC-Sim at all. It sounds like you want to only use MoorDyn to analyze the tension of the cable. For this, I would recommend consulting the [MoorDyn Documentation](https://moordyn.readthedocs.io/en/latest/#) and posting a thread on [MoorDyn's GitHub Repository](https://github.com/FloatingArrayDesign/MoorDyn) for support.
